### DISCUSSION TITLE
Motor not working with rc transmitter, but with inav motor works

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav#8715] [Q&A]
Having problem at setup:
The motor is not working with rc transmitter. All other servos are working perfectly
The motor can be activated by the inav configurator SW
Also you can see that the throttle channel is working at the screen, but the motor itself is doing nothing

Inav 5.1, Matek F405 WMN, with latest SW

The failsafe symbol is RED. All others are blue

Having searched the complete setup menu, but up to now haven't found any hint for this?

Could anybody please help

Thank you
 

**PLEASE MAKE SURE YOU READ AND UNDERSTAND THE SOCIAL MEDIA SUPPORT CHANNELS. QUESTIONS ABOUT FLASHING, CONFIGURING, PILOTING MAY BE CLOSED WITHOUT FURTHER INTERACTION.**

* [INAV Discord Server](https://discord.gg/peg2hhbYwN)
* [INAV Official on Facebook](https://www.facebook.com/groups/INAVOfficial)
* [INAV Official on Telegram](https://t.me/INAVFlight)

**Please double-check that nobody reported the issue before by using search in this bug tracker.**

**PLEASE DELETE THE TEXT ABOVE AFTER READING AND UNDERSTANDING IT**

****

## Steps to Reproduce

1.
2.
3.
4.

## Additional context

---

- FC Board name and vendor:
- INAV version string:

**Comment by MrD-RC** [comment]
INAV believes your receiver is in Failsafe. You need to check the failsafe setup of your receiver. You can't really progress until this is sorted out. You want that failsafe icon to be grey when your transmitter and receiver are powered.

**Comment by ghost** [comment]
Thanks for your reply.
The question is where does this effect come from? If the receiver would be really in fail save, all servos would go to special position (I have also connected 4 single servos for Roll, Pitch and Throttle to it. But this is not the case. Its working all normal. Also I would get an warning issue by the Jeti DC 16. But this is also  not the case. This effect is with all protocols SBUS and JETIEXBUS
